lambda/main/cortex.js

```diff
@@ -183,21 +183,21 @@ Cortex.prototype.getItemBySku = function (sku) {
 }
 
 /**
- * Will query keyword cortex resource
+ * Will query offersearch cortex resource
  * @param  {[String]} keyword        - The keyword to be searched
  * @return {[Promise]} - Returns a promise
  */
 Cortex.prototype.getItemsByKeyword = function (keyword) {
     return new Promise((resolve, reject) => {
-        this.cortexGet(`${this.cortexBaseUrl}?zoom=searches:keywordsearchform:itemkeywordsearchaction`)
+        this.cortexGet(`${this.cortexBaseUrl}?zoom=searches:offersearchform:offersearchaction`)
```

Comment:
I'm wondering if we should later do what store does and invoke a quick request to see if `offersearchform` is available, and fallback to `keywordsearchform` if it's not.

Reply:
That is a really good idea, I wasn't thinking about backwards compatibility.  I will update the issue.